muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/announcer.js

```diff
@@ -533,7 +577,7 @@
           } else {
             renderDustTemplate('announcer/announcer_items.dust',result,$.proxy(function (text) {
               var element = $(text);
-              $('.an-announcements-view-container').append(element);
+              $('.an-announcements-view-container').html(element);
```

Comment:
HTML takes string as input, the docs don't mention what happens when a jQuery element given. I'd rather use documented behavior.